lib/rucio/client/downloadclient.py

```diff
@@ -536,6 +538,8 @@ def _download_item(self, item, trace, log_prefix=''):
         :returns: dictionary with all attributes from the input item and a clientState attribute
         """
         logger = self.logger
+        pcache = Pcache()
```

Comment:
`pcache = Pcache() if self.check_pcache and len(item.get('archive_items', [])) == 0 else None`

Reply:
Let's do it like that for now, but that perhaps contradicts the main motivation for pcaches.